e2e-tests/chopsticks-configs/laos.yml

```diff
@@ -0,0 +1,17 @@
+endpoint: wss://rpc.laosvenus.gorengine.com
```

Comment:
What if we use the Laos local network instead, as we did previously? AFAIK, I think there is no need to start from the Venus network.